### ISSUE TITLE
[BUG] `Session.wait()` does not exit

### ISSUE THREAD

**Issue by benjaminpkane** [voxel51/fiftyone#1897]
On `v0.16.*`, `Session.wait()` does not exit after closing all App connections, which is the expected behavior.

```py
import fiftyone as fo

session = fo.launch_app()
session.wait(1)

# closing the app should exit this in a script
```

**Comment by PaRowsome** [comment]
Hey, just wondering if there are any updates on this issue? @benjaminpkane

**Comment by benjaminpkane** [comment]
Hi @PaRowsome. Apologies for missing this. Fixed in `v0.17.1`
